app/uk/gov/hmrc/play/language/LanguageController.scala

```diff
@@ -55,16 +61,19 @@ abstract class LanguageController @Inject()(implicit val messagesApi: MessagesAp
     * @return Redirect to referrer or fallbackURL, with new language. Or fallbackURL with default lang.
     */
 
+
+  val policy: RedirectUrlPolicy[Id] = OnlyRelative | PermitAllOnDev(environment)
```

Comment:
Please move this above the comment block as the comment block refers to the method below, switchToLanguage.
Please add to the comment block explaining that it the URL in the request header must match the desired RedirectUrlPolicy and that the policy can be overridden.
Can you also put a comment on the policy val explaining this can be overrided to match the needs of the application (in the same format as the vals above).

Reply:
applied comment